[2024-03-26T03:40:52.972+0000] {taskinstance.py:1979} INFO - Dependencies all met for dep_context=non-requeueable deps ti=<TaskInstance: upload_historical_congress_data.bq_create_member_Congress_partitioned_table_task scheduled__2024-03-15T00:00:00+00:00 [None]>
[2024-03-26T03:40:52.980+0000] {taskinstance.py:1979} INFO - Dependencies all met for dep_context=requeueable deps ti=<TaskInstance: upload_historical_congress_data.bq_create_member_Congress_partitioned_table_task scheduled__2024-03-15T00:00:00+00:00 [None]>
[2024-03-26T03:40:52.982+0000] {taskinstance.py:2193} INFO - Starting attempt 4 of 6
[2024-03-26T03:40:52.992+0000] {taskinstance.py:2217} INFO - Executing <Task(BigQueryInsertJobOperator): bq_create_member_Congress_partitioned_table_task> on 2024-03-15 00:00:00+00:00
[2024-03-26T03:40:52.998+0000] {standard_task_runner.py:60} INFO - Started process 108 to run task
[2024-03-26T03:40:53.003+0000] {standard_task_runner.py:87} INFO - Running: ['***', 'tasks', 'run', 'upload_historical_congress_data', 'bq_create_member_Congress_partitioned_table_task', 'scheduled__2024-03-15T00:00:00+00:00', '--job-id', '119', '--raw', '--subdir', 'DAGS_FOLDER/upload_historical_congress_data.py', '--cfg-path', '/tmp/tmpcq93gw_s']
[2024-03-26T03:40:53.006+0000] {standard_task_runner.py:88} INFO - Job 119: Subtask bq_create_member_Congress_partitioned_table_task
[2024-03-26T03:40:53.074+0000] {task_command.py:423} INFO - Running <TaskInstance: upload_historical_congress_data.bq_create_member_Congress_partitioned_table_task scheduled__2024-03-15T00:00:00+00:00 [running]> on host 2d1395ed6407
[2024-03-26T03:40:53.150+0000] {taskinstance.py:2513} INFO - Exporting env vars: AIRFLOW_CTX_DAG_EMAIL='***@***.com' AIRFLOW_CTX_DAG_OWNER='***' AIRFLOW_CTX_DAG_ID='upload_historical_congress_data' AIRFLOW_CTX_TASK_ID='bq_create_member_Congress_partitioned_table_task' AIRFLOW_CTX_EXECUTION_DATE='2024-03-15T00:00:00+00:00' AIRFLOW_CTX_TRY_NUMBER='4' AIRFLOW_CTX_DAG_RUN_ID='scheduled__2024-03-15T00:00:00+00:00'
[2024-03-26T03:40:53.154+0000] {crypto.py:82} WARNING - empty cryptography key - values will not be stored encrypted.
[2024-03-26T03:40:53.156+0000] {connection.py:269} WARNING - Connection schemes (type: google_cloud_platform) shall not contain '_' according to RFC3986.
[2024-03-26T03:40:53.157+0000] {base.py:83} INFO - Using connection ID 'google_cloud_default' for task execution.
[2024-03-26T03:40:53.220+0000] {bigquery.py:2811} INFO - Executing: {'query': {'query': 'CREATE OR REPLACE TABLE Congress.member         PARTITION BY DATE(updateDate)         AS         SELECT * FROM member_external_table;', 'useLegacySql': False}}'
[2024-03-26T03:40:53.223+0000] {bigquery.py:1613} INFO - Inserting job ***_upload_historical_congress_data_bq_create_member_Congress_partitioned_table_task_2024_03_15T00_00_00_00_00_3ee0cd924762bae1fbf30e40e7f0cf1c
[2024-03-26T03:40:54.015+0000] {taskinstance.py:2731} ERROR - Task failed with exception
Traceback (most recent call last):
  File "/home/airflow/.local/lib/python3.8/site-packages/airflow/models/taskinstance.py", line 444, in _execute_task
    result = _execute_callable(context=context, **execute_callable_kwargs)
  File "/home/airflow/.local/lib/python3.8/site-packages/airflow/models/taskinstance.py", line 414, in _execute_callable
    return execute_callable(context=context, **execute_callable_kwargs)
  File "/home/airflow/.local/lib/python3.8/site-packages/airflow/providers/google/cloud/operators/bigquery.py", line 2870, in execute
    job.result(timeout=self.result_timeout, retry=self.result_retry)
  File "/home/airflow/.local/lib/python3.8/site-packages/google/cloud/bigquery/job/query.py", line 1595, in result
    do_get_result()
  File "/home/airflow/.local/lib/python3.8/site-packages/google/cloud/bigquery/job/query.py", line 1584, in do_get_result
    super(QueryJob, self).result(retry=retry, timeout=timeout)
  File "/home/airflow/.local/lib/python3.8/site-packages/google/cloud/bigquery/job/base.py", line 971, in result
    return super(_AsyncJob, self).result(timeout=timeout, **kwargs)
  File "/home/airflow/.local/lib/python3.8/site-packages/google/api_core/future/polling.py", line 261, in result
    raise self._exception
google.api_core.exceptions.BadRequest: 400 Table "member_external_table" must be qualified with a dataset (e.g. dataset.table).; reason: invalid, location: member_external_table, message: Table "member_external_table" must be qualified with a dataset (e.g. dataset.table).

Location: US
Job ID: airflow_upload_historical_congress_data_bq_create_member_Congress_partitioned_table_task_2024_03_15T00_00_00_00_00_3ee0cd924762bae1fbf30e40e7f0cf1c

[2024-03-26T03:40:54.028+0000] {taskinstance.py:1149} INFO - Marking task as UP_FOR_RETRY. dag_id=upload_historical_congress_data, task_id=bq_create_member_Congress_partitioned_table_task, execution_date=20240315T000000, start_date=20240326T034052, end_date=20240326T034054
[2024-03-26T03:40:54.040+0000] {standard_task_runner.py:107} ERROR - Failed to execute job 119 for task bq_create_member_Congress_partitioned_table_task (400 Table "member_external_table" must be qualified with a dataset (e.g. dataset.table).; reason: invalid, location: member_external_table, message: Table "member_external_table" must be qualified with a dataset (e.g. dataset.table).

Location: US
Job ID: airflow_upload_historical_congress_data_bq_create_member_Congress_partitioned_table_task_2024_03_15T00_00_00_00_00_3ee0cd924762bae1fbf30e40e7f0cf1c
; 108)
[2024-03-26T03:40:54.055+0000] {local_task_job_runner.py:234} INFO - Task exited with return code 1
[2024-03-26T03:40:54.083+0000] {taskinstance.py:3312} INFO - 0 downstream tasks scheduled from follow-on schedule check
